packages/taquito/test/estimate/rpc-estimate-provider.spec.ts

```diff
@@ -563,9 +564,47 @@ describe('RPCEstimateProvider test signer', () => {
         storageLimit: 66,
       });
 
-      done();
-    });
-  });
+      done()
+    })
+    it('should throw an error with malformed source', async (done) => {
```

Comment:
minor nitpick, 'invalid' instead of malformed might be more straight forward here